spring-cloud-data-rest-client/src/main/java/org/springframework/cloud/data/rest/client/CloudDataTemplate.java

```diff
@@ -51,7 +52,12 @@
 	 */
 	private final CounterTemplate counterTemplate;
 
+	/**
+	 * REST client template for task operations.
+	 */
+	private final TaskTemplate taskTemplate;
```

Comment:
This will be very important given the testing plan @pperalta and I just started discussing... basically we will want to facilitate using mocks of the *Operations interfaces.